### PR TITLE
[Docs] Omit terminal prompt when using copy button

### PR DESCRIPTION
**Pull request by shrekris-anyscale**
Signed-off-by: Shreyas Krishnaswamy <shrekris@anyscale.com>

## Why are these changes needed?

When users copy documentation text with the copy button, they copy all the text in the code block. This change omits the terminal prompt `$` when copying CLI examples.

Example doc link: https://ray--28028.org.readthedocs.build/en/28028/serve/production-guide/rest-api.html#using-a-remote-cluster

* Copy-pasting the CLI command omits the `$` prompt:

Screen Shot 2022-08-20 at 3 43 22 PM

## Related issue number

N/A

## Checks

- [X] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [X] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - This change only affects doc formatting.

### CODE REVIEW

**Review by simon-mo — APPROVED** [review]

**Review by architkulkarni — APPROVED** [review]
Nice!!

**Comment by architkulkarni** [comment]
Linkcheck flaky on master and unrelated "403 Client Error: Forbidden for url: https://www.kdnuggets.com/2019/11/speed-up-pandas-4x.html"

**Review by edoakes — APPROVED** [review]

**Comment by edoakes** [comment]
Need codeowner approval

**Review by richardliaw — APPROVED** [review]

**Pull request merged**
